components/BrandingContent/ColorSection/ColorSection.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Section from 'components/_common_/Section/Section';
+import ColorBlock from './ColorBlock/ColorBlock';
+import styles from './colors.css';
+
+function ColorSection() {
+  return (
+    <Section title="Colors" theme="grey">
+      <h3>Primary Pallete</h3>
+      <div className={styles.colorPalette}>
+        <ColorBlock colorName="Blue" colorHex="#249CBC" />
```

Comment:
You can do `import { brandColorsObject } from 'common/styles/styleExports';` and have access to an object that looks like this:

```javascript
{
  "primary": "#249cbc",
  "secondary": "#d1665a",
  "mist": "#f0f2f2",
  "slate": "#47566b",
  // ... etc.
}
```

I advise important that object and thinking about how you can use your `<ColorBlock>` component in conjunction so that this brand page stays up-to-date even if our CSS variables change. You won't need to distinguish between "Tints" and "Primary" palette here. Just "Primary", "Secondary", and all the others.